src/pages/preferences/ProfileTab.tsx

```diff
@@ -25,6 +34,39 @@ export const ProfileTab: React.FC<ProfileTabProps> = ({
   const [timezoneField, setTimezoneField] = useState(timezone ?? "UTC");
   const [regionField, setRegionField] = useState(region ?? "us-east-1");
   const [githubUsernameField, setGithubUsernameField] = useState(lastKnownAs);
+  const dispatchBanner = useBannerDispatchContext();
+  const [updateUserSettings, { loading }] = useMutation<
+    UpdateUserSettingsMutation,
+    UpdateUserSettingsMutationVariables
+  >(UPDATE_USER_SETTINGS, {
+    onCompleted: () => {
+      dispatchBanner.successBanner(
+        `Your changes have successfully been saved.`
+      );
+    },
+    onError: (err) => {
+      dispatchBanner.errorBanner(`Error while saving patch: '${err.message}'`);
```

Comment:
typo: error while saving patch

Reply:
copied the error message from the other dispatch banner 😅

Reply:
should have paid attention to the contents